app/controllers/movie_actors_controller.rb

```diff
@@ -0,0 +1,8 @@
+class MovieActorsController < ApplicationController
+  def create
+    @movie_actor = MovieActor.new(movie_id: params[:id], actor_id: params[:actor])
+    if @movie_actor.save
+      redirect_to "/movies/#{params[:id]}"
+    end
+  end
+end
```

Comment:
In this controller, while the implementation is good, it will only work if the user enters an actor ID into the form, if they enter an actor's name, this code won't work. See my comments in your movie's show view.